lib/engine/game/g_1893/corporation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../../corporation'
+
+module Engine
+  module Game
+    module G1893
+      class Corporation < Engine::Corporation
+        def floated?
+          @name == 'AdSK' || super
```

Comment:
what do you mean that adsk is bonds? what does that mean?

Reply:
AdSK is the short name of the bonds used in 1893. They have a set price and revenue, and is bought/sold as shares.